<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSTModelCreator</class>
 <widget class="QMainWindow" name="SSTModelCreator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1141</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SST Model Creator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="1,1,10,1" columnstretch="1,1,2,1,2,2">
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap>sst-logo-small.png</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="2" alignment="Qt::AlignRight">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Enter Model Name --&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QPlainTextEdit" name="info">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCommandLinkButton" name="templates">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;This button will create your model templates (model.cc,model.h,simpleSimulation.cc,Makefile.am, and test_model.py The files will be placed in a  model directory in your working directory&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Generate/Open Templates</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" alignment="Qt::AlignRight">
       <widget class="QCheckBox" name="clean">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;If this box is checked a make clean will be run before make all install in the Configure step&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Make Clean</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QCommandLinkButton" name="Configure">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;This button will copy the model directory to the proper place inside the SST Framework then run autogen, configure and make all install&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Configure </string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" alignment="Qt::AlignLeft">
       <widget class="QCheckBox" name="autoRun">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;If this box is checked it will proceed directly to the Run SST step without having to select the button on the GUI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Run After Build</string>
        </property>
       </widget>
      </item>
      <item row="3" column="5">
       <widget class="QCommandLinkButton" name="RunSST">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;This button will run your new model inside the SST Framework by running sst model/tests/test_model.py&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Run SST</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="2">
       <widget class="QLineEdit" name="modelName">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Type in the name of your new model. The tool will not proceed if this box is empty.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QCheckBox" name="overwrite">
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;If this box is checked any model with the same name will be overwritten&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Overwrite Existing Model</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignRight">
       <widget class="QCommandLinkButton" name="browse">
        <property name="text">
         <string>Browse Templates</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QLineEdit" name="templateType">
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1141</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionOpen_Help"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#000000;&quot;&gt;Help Button will display the README file in the information screen.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_Help"/>
  </widget>
  <action name="actionOpen_Help">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
